eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1253,7 +1255,7 @@ class Channel(val nodeParams: NodeParams, wallet: EclairWallet, remoteNodeId: Pu
       goto(SHUTDOWN) using d.copy(commitments = commitments1)
 
     case Event(_: ChannelReestablish, d: DATA_NEGOTIATING) =>
-      forwarder ! d.localClosingSigned
+      forwarder ! d.closingTxProposed
```

Comment:
Is this correct? `d.closingTxProposed` is not a `LightningMessage`.

Reply:
It's not! adding a test...